02-express-tutorial/.gitignore

```diff
@@ -1 +1,2 @@
-/node_modules
\ No newline at end of file
+/node_modules
+.idea/
```

Comment:
This is good.  I almost always ignore .idea and .vscode when I bootstrap projects.